.vscodeignore

```diff
@@ -7,6 +7,8 @@
 !out/datascience-ui/viewers/commons.initial.bundle.js.map
 !out/datascience-ui/viewers/dataExplorer.js.map
 !out/datascience-ui/viewers/plotViewer.js.map
+images/**
```

Comment:
What are these for?

Reply:
Ahh, nevermind I see the comment.

Reply:
We appear to be shipping https://github.com/microsoft/vscode-jupyter/tree/main/images I don't think we want that?

Reply:
Yup, I just didn't see your initial commit comment on removal. Wasn't sure how it related to smoke tests. Makes sense now.